language/python/build-images.md

```diff
@@ -96,10 +105,10 @@ Before we can run `pip3 install`, we need to get our `requirements.txt` file int
 COPY requirements.txt requirements.txt
 ```
 
-Once we have our `requirements.txt` file inside the image, we can use the `RUN` command to execute the command `pip3 install`. This works exactly the same as if we were running `pip3 install` locally on our machine, but this time the modules are installed into the image.
+Once we have our `requirements.txt` file inside the image, we can use the `RUN` command to execute the command `python3 -m pip install`. This works exactly the same as if we were running `python3 -m pip install` locally on our machine, but this time the modules are installed into the image.
 
 ```dockerfile
-RUN pip3 install -r requirements.txt
+RUN python3 -m pip install -r requirements.txt
```

Comment:
I'm not a day to day Python developer, but this looks a bit weird to me as inside a container `pip3` is installed.
I understand that you want to have it consistent to the commands to run on the host, but do Python developers actually run pip this way? Just asking as I'm curious 😃